### ISSUE TITLE
Release archive 5.1 does not build convertf nor mergeit  

### ISSUE THREAD

**Issue by EricDeveaud** [DReichLab/AdmixTools#45]
hello,

releaase archive does not build convertf nor mergeit while git tree will.

after issuing make all and make install in both git cloned tree and untared release-v5.1 content of bin directories differs

# git clone bin directory 
```
bigmess:/tmp/AdmixTools > \ls bin 
convertf  expfit.sh  qp3Pop   qpAdm    qpDpart  qpF4ratio  qpWave     qpreroot   wtjack.pl
dowtjack  mergeit    qp4diff  qpBound  qpDstat  qpGraph    qpff3base  rexpfit.r
```

# untared release archive.
```
bigmess:/tmp/AdmixTools-5.1 > \ls bin      
dowtjack   qp3Pop   qpAdm    qpDpart  qpF4ratio  qpWave     qpreroot   wtjack.pl
expfit.sh  qp4diff  qpBound  qpDstat  qpGraph    qpff3base  rexpfit.r
```

easy to fix, because it's just a deifference in src/Makefile, see

```
-- /tmp/AdmixTools-git/src/Makefile    2018-11-16 10:42:52.830661146 +0100
+++ /ltmp/AdmixTools-5.1/src/Makefile  2018-09-27 18:27:42.000000000 +0200
@@ -21,7 +21,7 @@
 ND = nicksrc
 NLIB = $(ND)/libnick.a
 
-PROGS= qp3Pop qpDstat qpF4ratio qpAdm qpWave qp4diff dowtjack expfit.sh  qpBound qpGraph qpreroot qpff3base qpDpart convertf mergeit ##  gcount kimf
+PROGS= qp3Pop qpDstat qpF4ratio qpAdm qpWave qp4diff dowtjack expfit.sh  qpBound qpGraph qpreroot qpff3base qpDpart ##  gcount kimf
 ## rolloff* convertf mergeit from .../o2src
 
 PROGS2 = rexpfit.r  wtjack.pl 
```

**Comment by MatthewMah** [comment]
This is already fixed in 213ceff8f2bf4cde6fdfe304b622ff0f2ef778ae.

**Comment by EricDeveaud** [comment]
yep.

but you may want to consider that people installing software and or packaging software tend to work form release tag instead of pulling the repo.
we istall software on a cluster and try to stcik to release tags. 
reasons underneath are mainly driven by reproducibiliyty concerns and or automatic packaging.

please either fix the 5.1 release or issue a 5.1.1, 5.2 version 

regards
